yt/frontends/_skeleton/__init__.py

```diff
@@ -1,14 +1,14 @@
 """
-API for yt.frontends.skeleton
+API for yt.frontends._skeleton
 
 
 
 """
 
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 # Copyright (c) 2013, yt Development Team.
 #
 # Distributed under the terms of the Modified BSD License.
 #
 # The full license is in the file COPYING.txt, distributed with this software.
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
```

Comment:
The changes you've made to the copyright header should be reverted I think, that should match the rest of the codebase, and should only be changed in a codebase-wide change.